index.js

```diff
@@ -39,31 +39,31 @@ config = rc(pjson.name, defaultConfig);
             process.exit(1);
         }
 
-        const backupCode = `${config.autoUpdate.backupDirectory}/AutoGitUpdate/backup`;
+        const backupCode = `${config.autoUpdate.backupDirectory}/auto-update/backup`;
```

Comment:
Please rename the variable to backupCodeDirectory and create path using path.join. Also if i see it correctly you are creating two same variables - backupCode and source, use only one and delete the other.